lib/bolt/outputter.rb

```diff
@@ -14,6 +14,20 @@ def self.for_format(format)
     def initialize(stream = $stdout)
       @stream = stream
     end
+
+    # Expects to be called with a configured Puppet compiler or error.instance? will fail
```

Comment:
This is annoying, but not sure what to do about it, and it'll require work in Puppet to fix. /cc @thallgren

Reply:
@MikaelSmith , please create a ticket for this and assign it to me. It will be easy to fix.

Reply:
Can't we just unwrap this inside run_plan instead of the outputter? It seems cleaner if we're converting from puppety stuff to bolty stuff to do it there instead of where we "output" the value anyway.

Reply:
Ah, I'll do that. I guess we need to define what print_plan expects if not an ExecutionResult.

Reply:
Done. That turned out a lot cleaner.